.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+dist: xenial
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.5
+    - python: 3.6
+    - python: 3.7
```

Comment:
Based on my sami2py streamlined setup.  We can pull the other tests out once we get Travis CI set up if needed.  Tests take about 2 min run locally.